.github/workflows/ci.yml

```diff
@@ -5,13 +5,15 @@ on:
     paths-ignore: # don't run this workflow if it only contains v3 changes
       - 'v3/**'   # https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#example-excluding-paths
       - '.github/workflows/v3.yml'
+    tags-ignore:
+      - '3.*'
```

Comment:
Can we also ignore the `main` branch? I don't think we ever want to trigger a v2 workflow on the `main` branch, nor do we ever want to trigger a v3 workflow on the `master` branch.